app/story/utility.py

```diff
@@ -7,7 +7,6 @@
 from app.models.userdata_models import Location
 from app.models import utility
 
-
```

Comment:
And thy speakest: Thouh shall leave two newlines below import statements

Reply:
ok ^^